Packs/cyberark_AIM/ReleaseNotes/1_0_19.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+
+##### CyberArk AIM v2
+Updated the Docker image to: *demisto/ntlm:1.0.0.106849*.
```

Comment:
@adi88d - Please add a breaking change note as well for this Docker update. See https://jira-dc.paloaltonetworks.com/browse/XSUP-36507.

Reply:
done